### PR TITLE
Support animated gif in entry list

### LINKED ISSUE #289: We need to support animated GIF's
Using licecap or silentcast it is easy to make animated GIF's. When images are uploaded to django though they are resized and converted to PNG. We need to update the logic so thumbs etc. can be created for animate GIF's without losing the animation.

### PR DESCRIPTION
**Pull request by dimasciput**
Fix #289

### CODE REVIEW

**Comment by timlinux** [comment]
Can you fix the flake error so we can merge?

```
1.19s$ flake8 --config .flake8 django_project
django_project/base/templatetags/custom_markup.py:26:1: W391 blank line at end of file
```

File: django_project/base/templatetags/custom_markup.py [hunk 1]
```diff
@@ -16,3 +17,10 @@ def base_markdown(value):
                                        extensions,
                                        safe_mode=True,
                                        enable_attributes=False))
+
+
+@register.filter(name='is_gif', is_safe=True)
+@stringfilter
+def is_gif(value):
+    return bool(re.search('.gif', value))
```
Comment (timlinux) [inline django_project/base/templatetags/custom_markup.py]: Probably we should terminate the regex to match end of phrase only...

**Comment by dimasciput** [comment]
@timlinux updated and fixed the error

**Comment by timlinux** [comment]
Thanks @dimasciput LGTM

**Pull request merged**